tools/events-automation/README.md

```diff
@@ -1 +1,71 @@
-TODO: write this
\ No newline at end of file
+### Event Class:
+**Required Variables**
+- name (string): Title of the event.
```

Comment:
Non-blocking nit: each variable name can be codeblocked by using back-ticks, e.g.
`name`

Reply:
```
`name`
```